objectclient/object_client.go

```diff
@@ -248,7 +248,7 @@ func (p *ObjectClient) Watch(opts metav1.ListOptions) (watch.Interface, error) {
 	}
 	streamDecoder := streaming.NewDecoder(json2.Framer.NewFrameReader(r), embeddedDecoder)
 	decoder := restclientwatch.NewDecoder(streamDecoder, embeddedDecoder)
-	return watch.NewStreamWatcher(decoder), nil
+	return watch.NewStreamWatcher(decoder, nil), nil
```

Comment:
what is the type of this second arg?

Reply:
ErrorReporter. I wasn't sure what msg and error code to return it with, so for now kept it nil in proxy_store and objectclient 
https://github.com/kubernetes/kubernetes/pull/73937/files